fedora/base/source.yaml

```diff
@@ -0,0 +1,9 @@
+---
+apiVersion: cdi.kubevirt.io/v1beta1
+kind: VolumeImportSource
+metadata:
+  name: fedora
+spec:
+  source:
+    registry:
+      url: "docker://quay.io/nirsof/dr-demo:fedora-6"
```

Comment:
Would be better to have the fedora version as the image name?

Reply:
I can use fedora-40-6 - 6 is the release number. But lets do this later to get this repo with some content now.